client/informant/src/lib.rs

```diff
@@ -49,62 +43,15 @@ impl Default for OutputFormat {
 	}
 }
 
-/// Marker trait for a type that implements `TransactionPool` and `MallocSizeOf` on `not(target_os = "unknown")`.
-#[cfg(target_os = "unknown")]
-pub trait TransactionPoolAndMaybeMallogSizeOf: TransactionPool {}
-
-/// Marker trait for a type that implements `TransactionPool` and `MallocSizeOf` on `not(target_os = "unknown")`.
-#[cfg(not(target_os = "unknown"))]
-pub trait TransactionPoolAndMaybeMallogSizeOf: TransactionPool + MallocSizeOf {}
-
-#[cfg(target_os = "unknown")]
-impl<T: TransactionPool> TransactionPoolAndMaybeMallogSizeOf for T {}
-
-#[cfg(not(target_os = "unknown"))]
-impl<T: TransactionPool + MallocSizeOf> TransactionPoolAndMaybeMallogSizeOf for T {}
-
 /// Builds the informant and returns a `Future` that drives the informant.
 pub fn build<B: BlockT, C>(
 	client: Arc<C>,
-	network_status_sinks: Arc<status_sinks::StatusSinks<NetworkStatus<B>>>,
-	pool: Arc<impl TransactionPoolAndMaybeMallogSizeOf>,
-	format: OutputFormat,
 ) -> impl futures::Future<Output = ()>
 where
 	C: UsageProvider<B> + HeaderMetadata<B> + BlockchainEvents<B>,
 	<C as HeaderMetadata<B>>::Error: Display,
 {
-	let mut display = display::InformantDisplay::new(format.clone());
-
-	let client_1 = client.clone();
-	let (network_status_sink, network_status_stream) = tracing_unbounded("mpsc_network_status");
-	network_status_sinks.push(Duration::from_millis(5000), network_status_sink);
-
-	let display_notifications = network_status_stream
-		.for_each(move |net_status| {
-			let info = client_1.usage_info();
-			if let Some(ref usage) = info.usage {
-				trace!(target: "usage", "Usage statistics: {}", usage);
-			} else {
-				trace!(
-					target: "usage",
-					"Usage statistics not displayed as backend does not provide it",
-				)
-			}
-			#[cfg(not(target_os = "unknown"))]
-			trace!(
-				target: "usage",
-				"Subsystems memory [txpool: {} kB]",
-				parity_util_mem::malloc_size(&*pool) / 1024,
-			);
-			display.display(&info, net_status);
-			future::ready(())
-		});
-
-	future::join(
-		display_notifications,
-		display_block_import(client),
-	).map(|_| ())
```

Comment:
I don't really remember how it works, but it's definitely wrong to remove all this code.
This code is what displays every 5 seconds the line that looks like `⚙️  Syncing, target=#5506013 (11 peers), best: #1492314 (0x3936…6835), finalized #1416901 (0x843c…aa61), ⬇ 176.1kiB/s ⬆ 9.5kiB/s`.

Reply:
I reverted the code